guide/src/telemetry/operators.md

```diff
@@ -0,0 +1,100 @@
+# Hermes operators guide to using metrics
+
+This section is a basic guide on how Hermes metrics can be used to observe both
+the current state of the Hermes relayer and the networks it is connected to.
+
+## General remarks about the metrics
+
+- All Hermes metrics are tracked and updated from the moment the Hermes service (i.e., `start`) starts up.
+Metrics are automatically reset if the service is restarted.
+- For maximum reliability, it is advised to combine monitoring of your Hermes service with monitoring of your full nodes.
+- TODO: Grafana template here.
```

Comment:
FYI @AlianBenabdallah this is likely where we're reference a Grafana template.

Reply:
I think we can add Grafana templates in another PR